### DISCUSSION TITLE
Rename master to main

### DISCUSSION THREAD

**Discussion by vasilvestre** [artandor/multistream-tools#33] [Ideas]
As it is the new standard, I see no reason to do not rename it.

**Comment by artandor** [comment]
I think i did not because symfonycloud would not work with main, and some other tools aswell. But it seems that we won't need those tools at all ?

**Reply by vasilvestre** [reply]
I remember this, I think we can move safely so. Only the administrator can change the default github branch and I can't :)

**Comment by artandor** [comment]
Done
